mod-source-record-storage-server/src/main/java/org/folio/services/handlers/actions/AbstractUpdateModifyEventHandler.java

```diff
@@ -184,6 +185,8 @@ private void preparePayload(DataImportEventPayload dataImportEventPayload) {
 
   private void increaseGeneration(Record changedRecord) {
     var generation = changedRecord.getGeneration();
-    changedRecord.setGeneration(isNull(generation) ? 1 : ++generation);
+    if(nonNull(generation)){
```

Comment:
What will happen if generation is NULL?

Reply:
Nothing, just like here org.folio.services.handlers.InstancePostProcessingEventHandler#sendEventToDataImportLog

This method was added for in MODSOURMAN-692